docs/pfcwd/README_PFCWD_Restart.md

```diff
@@ -0,0 +1,87 @@
+# Readme PFCWD  Restart 
+- [Readme PFCWD  Restart](#readme-pfcwd-restart)
+  - [Background](#background)
+    - [PFC storm detection](#pfc-storm-detection)
+    - [PFC storm mitigation](#pfc-storm-mitigation)
+    - [PFC storm restoration](#pfc-storm-restoration)
+    - [PFC watchdog implementation](#pfc-watchdog-implementation)
+  - [PFC watchdog commands on SONiC](#pfc-watchdog-commands-on-sonic)
```

Comment:
These sections can be omitted. Please move all this to a general readme under the pfcwd folder since it is the same for all cases

Reply:
Omitted the TOC from the readme.